python/cudf/cudf/core/column/string.py

```diff
@@ -1528,6 +1536,238 @@ def findall(self, pat, flags=0, **kwargs):
             cpp_findall(self._column, pat), **kwargs
         )
 
+    def isempty(self, **kwargs):
+        """
+        Check whether each string is a an empty string.
+
+        Returns : Series or Index of bool
+            Series or Index of boolean values with the same length as
+            the original Series/Index.
+        """
+        return self._return_or_inplace(
+            (self._parent == "").fillna(False), **kwargs
+        )
+
+    def isspace(self, **kwargs):
+        """
+        Check whether all characters in each string are whitespace.
+
+        Returns : Series or Index of bool
+            Series or Index of boolean values with the same length as
+            the original Series/Index.
+        """
+        return self._return_or_inplace(cpp_isspace(self._column), **kwargs)
+
+    def endswith(self, pat, na=np.nan, **kwargs):
```

Comment:
I don't think we should use `np.nan` as the default here even though that's what Pandas does. We have a true `null`, lets use it.

Reply:
In hindsight this only fires when a non-String object is passed, which is impossible for us. Suggest removing it entirely from the signature and then if it's in `**kwargs` we should just throw a warning that it's a non-issue for cuDF since it has true string types.

Reply:
@kkraus14, removed `na` and checking for it in `**kwargs`.